src/app/tests/suites/commands/commissioner/CommissionerCommands.cpp

```diff
@@ -48,11 +44,40 @@ CHIP_ERROR CommissionerCommands::PairWithManualCode(chip::NodeId nodeId, const c
 
 CHIP_ERROR CommissionerCommands::Unpair(chip::NodeId nodeId)
 {
-    mExpectedStatus = CHIP_NO_ERROR;
-
     return GetCurrentCommissioner().UnpairDevice(nodeId);
 }
 
+chip::app::StatusIB ConvertToStatusIB(CHIP_ERROR err)
+{
+    using namespace chip::Protocols::InteractionModel;
```

Comment:
Why not throw a `using namespace chip` and `using chip::app::StatusIB` in here and get rid of all the namespace noise in this method?

Reply:
Fixed.